h2o-algos/src/test/java/hex/generic/GenericModelTest.java

```diff
@@ -675,7 +677,97 @@ public void downloadable_mojo_glm() throws IOException {
             assertArrayEquals(FileUtils.readFileToByteArray(originalModelMojoFile), FileUtils.readFileToByteArray(genericModelMojoFile));
 
         } finally {
-        Scope.exit();
+            Scope.exit();
+        }
+    } 
+    
+    /**
+     * Create a CoxPH model and writes a MOJO into a temporary zip file. Then, it creates a Generic model out of that
+     * temporary zip file and re-downloads the underlying MOJO again. The byte arrays representing both MOJOs are tested
+     * to be the same.
+     *
+     */
+    @Test
+    public void downloadable_mojo_cox_ph() throws IOException {
```

Comment:
```suggestion
    public void downloadableMojoCoxPh() throws IOException {
```

We are in Java, please. :)

Reply:
This style goes through the whole GenericModelTest. I will add it in the PR to clean the test: 

https://github.com/h2oai/h2o-3/pull/5158/files#diff-d01f2061f9cc0c8a9e6f33a7a5e9c77df07a8094ba3dcbfe4205d0123a946139

Reply:
:+1: